bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -488,33 +511,26 @@ if onsource_file:
         trig_id = loud_on_bestnr_trigs
         trig_index = np.where(on_trigs['network/event_id'] == trig_id)[0][0]
         ifo_trig_index = {
-            ifo: np.where(on_trigs['%s/event_id' % ifo] == trig_id)[0][0]
+            ifo: np.where(on_trigs[ifo+'/event_id'] == trig_id)[0][0]
             for ifo in ifos
         }
         num_trials_louder = 0
-        tot_off_snr = np.array([])
-        for slide_id in slide_dict:
-            num_trials_louder += sum(time_veto_max_bestnr[slide_id] >
-                                     loud_on_bestnr)
-            tot_off_snr = np.concatenate([tot_off_snr,
-                                          time_veto_max_bestnr[slide_id]])
-        fap = num_trials_louder/total_trials
-        fap_test = sum(tot_off_snr > loud_on_bestnr)/total_trials
-        pval = '< %.3g' % (1./total_trials) if fap == 0 else '%.3g' % fap
+        fap = sum(sorted_bkgd > loud_on_bestnr)/total_trials
+        pval = f'< {(1./total_trials):.3g}' if fap == 0 else f'{fap:.3g}'
```

Comment:
I have seen this code a few times now. Maybe add "define a common `format_pvalue_str()` function" to the wishlist.